pkgs/games/xpilot/default.nix

```diff
@@ -1,7 +1,7 @@
-{stdenv, fetchurl, libX11, SDL, mesa, expat, SDL_ttf, SDL_image}:
+{stdenv, fetchurl, xorg, SDL, mesa, expat, SDL_ttf, SDL_image, zlib}:
 let
   buildInputs = [
-    libX11 SDL SDL_ttf SDL_image mesa expat
+    xorg.libX11 xorg.libSM SDL SDL_ttf SDL_image mesa expat zlib
```

Comment:
We are generally moving from using `xorg` to just `libX11`, `libSM` and friends.